resolwe/process/runtime.py

```diff
@@ -8,6 +9,15 @@
 
 try:
     from plumbum import local as Cmd  # pylint: disable=unused-import
+    # Log plumbum commands to standard output.
+    root = logging.getLogger('plumbum.local')
+    root.setLevel(logging.DEBUG)
+
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setLevel(logging.DEBUG)
+    formatter = logging.Formatter('%(asctime)s - %(message)s')
```

Comment:
Make this more verbose, to make it clear where the logs come from, e.g. `%(asctime)s - %(levelname)s - %(name)s[%(process)s]: %(message)s`.

Reply:
I kind-off wanted to avoid that in stdout.

Reply:
The crucial information for me is `%(name)s` to know where the log comes from.

Do you need `%(asctime)s`? Dows any other tool in our processes uses time in logs?

Reply:
Why would a bioinformatician that runs the process need to know where the log comes from?

Reply:
The time is useful because the order of commands is switched sometimes in stdout.

Reply:
> Why would a bioinformatician that runs the process need to know where the log comes from?

This is probably not useful now that there is only one source of logs, but it will be much more transparent when we add other sources, e.g. it would be nice to pipe all stdouts through logging.